File: 2-second-year-2022-2023/Programming-languages-and-laboratory/Python/Exercises/26_compito_java_20181119/Regolare.py
from Partecipante import Partecipante


class Regolare(Partecipante):
    def __init__(self, codice, tipo, nome, cognome, ente, dipendenti, accademico):
        super().__init__(codice, tipo, nome, cognome)
        self._ente = ente
        self._dipendenti = dipendenti
        self._accademico = accademico

    def get_ente(self):
        return self._ente

    def __str__(self):
        return super().__str__() + "\t-\t-\t-\t" + self._ente + "\t" + str(self._dipendenti) + "\t" + str(self._accademico)
        

File: II - Linguaggi di programmazione e laboratorio/Java/Exercises/26_compito_java_2019_11_19/Regolare.java
import java.util.*;

public class Regolare extends Partecipante {

	private int dip;
	private String ente;
	private boolean acc;

	public Regolare(int codice, String nome, String cognome, String ente, int dip, boolean acc) {
		super(codice, nome, cognome, "regolare");
		this.ente = ente;
		this.dip = dip;
		this.acc = acc;
	}

	public String getEnte() {
		return ente;
	}

	public String toString() {
		return super.toString() + "\t-\t-\t" + ente + "\t" + dip + "\t" + acc;
	}

}
